torch/distributed/_pipeline/sync/checkpoint.py

```diff
@@ -45,7 +45,7 @@
 
 # Types for shared memory between Checkpoint and Recompute.
 Recomputed = Tuple[TensorOrTensors, Tensors]  # (output, input_leaf)
-RNGStates = Tuple[ByteTensor, Optional[ByteTensor]]  # (cpu_rng_state, gpu_rng_state)
+RNGStates = Tuple[Tensor, Optional[Tensor]]  # (cpu_rng_state, gpu_rng_state)
```

Comment:
Both torch.get_rng_state() and torch.cuda.get_rng_state(device) returns Tensor, not ByteTensor (in function save_rng_states()). So RNGStates should accept Tensor, not ByteTensor.